contracts/interfaces/ILensHub.sol

```diff
@@ -226,6 +226,30 @@ interface ILensHub {
      */
     function followWithSig(DataTypes.FollowWithSigData calldata vars) external;
 
+    /**
+     * @notice Toggle Follows on the given profiles, emiting toggle event for each Follow profile.
+     *
+     * NOTE: `profileIds`, `followNFTIds` and `enables` arrays must be of the same length.
+     *
+     * @param profileIds The token ID array of the profiles to follow.
+     * @param followNFTIds The token ID array of the followNFT.
+     * @param enables The boolean indicates when the `follow` logic on the indexer needs to be enabled/disabled
+     */
+    
```

Comment:
unneeded linebreak
also, linter warns about trailing whitespaces around this function